docs/GOVERNANCE.md

```diff
@@ -0,0 +1,155 @@
+# Crystal governance
+
+## Abstract
+
+This text outlines the governance process for Crystal. Governance is based around a steering council. The council has broad authority, which they seek to exercise as rarely as possible.
+
+## The steering council
+
+### Composition
+
+The steering council is a committee formed ideally by an odd and small number of people: 3 or 5.
+
+### Current steering council
+
+The current steering council consists of:
+
+- Ary Borenszweig
+- Brian Cardiff
+- Juan Wajnerman
+
+### Mandate
+
+The steering council shall work to:
+
+1. Maintain the quality and stability of the Crystal language, compiler, std-lib and any other project under [crystal-lang organization](http://github.com/crystal-lang).
+
+2. Make contributing as accessible, inclusive, and sustainable as possible.
```

Comment:
`inclusive` as a word has become politically divisive in the past few years is there a way to choose other language for this?

Reply:
I am not saying the thought behind the sentiment is wrong but is there maybe a way to spell it out or something.

Reply:
Please make concrete constructive proposals, otherwise I don't see this as a discussion worth having.

Reply:
Suggestion; maybe we pick some language to defer to the standing [Code of Conduct](https://github.com/crystal-lang/crystal/blob/master/CODE_OF_CONDUCT.md) instead? I think that answers all questions as to what this bullet point stands for.

> Make contributing as accessible, inclusive, and sustainable as possible, as described by the [Code of Conduct](https://github.com/crystal-lang/crystal/blob/master/CODE_OF_CONDUCT.md)

or similar

Reply:
Since there is a code of conduct and other language like this I am going to remove my comment.